README.md

```diff
@@ -781,18 +780,18 @@ To copy, cut and paste in such cases (but only supported on Android >= 7):
  - <kbd>MOD</kbd>+<kbd>v</kbd> injects `PASTE` (after computer-to-device
    clipboard synchronization)
 
-In addition, <kbd>MOD</kbd>+<kbd>Shift</kbd>+<kbd>v</kbd> allows to inject the
-computer clipboard text as a sequence of key events. This is useful when the
-component does not accept text pasting (for example in _Termux_), but it can
-break non-ASCII content.
+In addition, <kbd>MOD</kbd>+<kbd>Shift</kbd>+<kbd>v</kbd> allows you to inject
```

Comment:
Is "allow injecting" also valid?

(refs https://github.com/Genymobile/scrcpy/pull/3174#discussion_r846769006)

Reply:
The words *"allow injecting"* wouldn't be grammatically correct here, but *"injects"* works and is simple.

Reply:
Ok for "injects" :+1: